addons/stock/models/product.py

```diff
@@ -432,6 +432,11 @@ class ProductTemplate(models.Model):
         relation="stock.location.route", string="Category Routes",
         related='categ_id.total_route_ids')
 
+    @api.depends(
+        'product_variant_ids',
+        'product_variant_ids.stock_quant_ids',
+        'product_variant_ids.stock_move_ids',
```

Comment:
I think this depends line is not needed at all

Reply:
Removed